sp-privacy/LICENSE

```diff
@@ -0,0 +1,845 @@
+sp-privacy
+Copyright © 2021 System1 and other contributors
+
+sp-privacy as a whole is presently Licensed GPL v3+, though many portions
+of the code are dual-licensed under other free/open source licenses.
+
+CONTRIBUTORS AGREE TO FREE/OPEN SOURCE DUAL-LICENSING
+=====================================================
+
+By contributing code or other works of authorship to this project ("Your
+Contributions"), you grant to the System1 a
+perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable
+license to reproduce, prepare derivative works of, publicly display, publicly
+perform, sublicense, and distribute Your Contributions and such derivative
+works under the terms of any free software or open source license or licenses,
+such license(s) to be selected by System1 at its sole discretion (such licenses
+will always be consistent with System1's mission).
+
+By contributing code or other works of authorship to this project, you
+represent that you have the legal right and ability to grant this license.
+
+Except for the license granted herein to System1 and recipients of software
+distributed by System1, You reserve all right, title, and interest in and to Your
+Contributions.
+
+If you do not agree to these terms, please do not send patches or pull
+requests, or commit any material to this repository.
+
+LICENSES FOR INCORPORATED CODEBASES
+===================================
```

Comment:
i wouldn't include all of these unless we actually pulled them into the privacy badger repo, only reference any 3rd parties we've added to this codebase